app/src/main/res/values-fr/strings.xml

```diff
@@ -11,8 +11,320 @@
+  <string name="submit_xform_failed_server_auth">Erreur forme soumettant: n\'a pas pu authentifier avec le serveur.</string>
+  <string name="submit_xform_failed_server_timeout">Timed out tout en envoyant formulaire au serveur.</string>
+  <string name="submitting_encounter_form_title">Envoi de forme</string>
+  <string name="submitting_encounter_form_message">S\'il vous plaît patienter pendant que les données du formulaire sont soumis.</string>
+  <string name="consciousness">Conscience</string>
+  <string name="condition">État</string>
+  <string name="cancel_sync">Annuler synchronisation</string>
+  <string name="cancelling_sync">Annulation de la synchronisation…</string>
+  <string name="sync_starting">Synchronisation partir…</string>
+  <string name="troubleshoot_wifi_disabled">Wifi est désactivé</string>
+  <string name="troubleshoot_wifi_disabled_action_enable">Activer</string>
+  <string name="troubleshoot_wifi_disconnected">Wifi est déconnecté</string>
+  <string name="troubleshoot_wifi_disconnected_action_connect">Connectez</string>
+  <string name="troubleshoot_server_auth">Serveur nom d\'utilisateur / mot de passe peut être incorrect</string>
+  <string name="troubleshoot_server_auth_action_check">Vérifier</string>
+  <string name="troubleshoot_server_address">Adresse du serveur peut être incorrect</string>
+  <string name="troubleshoot_server_address_action_check">Vérifier</string>
+  <string name="troubleshoot_server_unreachable">Serveur inaccessible</string>
+  <string name="troubleshoot_action_more_info">Plus d\'information</string>
+  <string name="troubleshoot_server_unreachable_details">
+      Le serveur n\'a pas pu être atteint. Cela peut être parce que:
+      \n
+      \n • Le réseau wifi est incorrecte.
+      \n • L\'URL du serveur est incorrecte.
+      \n • Le serveur est en panne.
+      \n
+      \n S\'il vous plaît contacter un administrateur.
+  </string>
+  <string name="troubleshoot_server_unstable">Server peut être instable</string>
+  <string name="troubleshoot_server_unstable_details">
+      Le serveur répond actuellement avec le code d\'erreur 500, indiquant que le serveur peut être dans un état d\'erreur.
+      \n
+      \n S\'il vous plaît contacter un administrateur.</string>
+  <string name="troubleshoot_server_not_responding">Serveur ne répond pas</string>
+  <string name="troubleshoot_server_not_responding_details">
+      Le serveur est actuellement ne répond pas aux demandes. Cela peut être parce que:\n
+      \n • Le serveur est momentanément indisponible.
+      \n • Le serveur est en cours de démarrage.
+      \n
+      \n S\'il vous plaît contacter un administrateur.</string>
+  <string name="patient_location_error_unknown">Une erreur inconnue est survenue lors de la mise à jour se trouve le patient. Veuillez réessayer.</string>
+  <string name="patient_location_error_interrupted">Patient mise à jour de l\'emplacement a été interrompu. Veuillez réessayer.</string>
```

Comment:
not sure. meaybe 'Mise à jour de l\'emplacement du patient a été interrompu. Veuillez réessayer.'